### DISCUSSION TITLE
Can I use Zod in TypeScript Older versions?

### DISCUSSION THREAD

**Discussion by JorgeDorio** [colinhacks/zod#2096] [Q&A] [answered]
I'm working in a project running in `node 12.22` and `TypeScript ^3.7.4`

in the readme it has specified `TypeScript 4.5+`, but I assume that is for the current version of Zod. Is there any release where I can use the tool in this version of typescript?

**Comment by JacobWeisenburger** [comment, marked as answer]
I very much do not recommend doing this. But if you have no other choice, then you might be able to use zod v2 or v1. More info can be found here: [Readme from last year](https://github.com/colinhacks/zod/blob/c1d0d00e37c8eee57ff789100715c3f99fb31308/README.md#typescript-requirements)
